File: src/data/types/itemBasePrice.ts
// Necessary completion: Agent 14's contract asks for a "Base price config
// (JSON): a base Credits price for each of the MVP's five items," but
// neither the Phase 3 amendment nor Agent 11 defined a shape for it --
// PlanetMarketState.basePrice is per planet+item, not the single
// galaxy-wide reference value each planet's initial state is seeded from.
export interface ItemBasePrice {
  itemId: string;
  basePrice: number;
}

File: unity/ProfitableCore/Schema/ItemBasePrice.cs
namespace Profitable.Core.Schema;

// Ports src/data/types/itemBasePrice.ts. The single galaxy-wide reference
// value each planet's initial PlanetMarketState.BasePrice is seeded from
// -- not the same thing as PlanetMarketState.BasePrice itself, which is
// per planet+item.
public class ItemBasePrice
{
    public string ItemId { get; set; } = string.Empty;
    public double BasePrice { get; set; }
}
